<ui version="4.0" >
 <class>MessageWindowClass</class>
 <widget class="QMainWindow" name="MessageWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MessageWindow</string>
  </property>
  <property name="iconSize" >
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="dockNestingEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <property name="leftMargin" >
     <number>2</number>
    </property>
    <property name="topMargin" >
     <number>2</number>
    </property>
    <property name="rightMargin" >
     <number>2</number>
    </property>
    <property name="bottomMargin" >
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="wdtTabs" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMessage" >
       <attribute name="title" >
        <string>Message</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>4</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>4</number>
        </property>
        <item>
         <widget class="QWidget" native="1" name="wdtInfo" />
        </item>
        <item>
         <widget class="QWidget" native="1" name="wdtReceivers" >
          <layout class="QHBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblReceiversLabel" >
             <property name="text" >
              <string>To:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblReceivers" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
             <property name="margin" >
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="lblSubject" >
            <property name="text" >
             <string>Subject:</string>
            </property>
            <property name="buddy" >
             <cstring>lneSubject</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lneSubject" />
          </item>
          <item>
           <widget class="QLabel" name="lblReceived" >
            <property name="text" >
             <string>Received:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDateTime" >
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" native="1" name="wdtMessage" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="wdtButtons" >
          <layout class="QHBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pbtNext" >
             <property name="text" >
              <string>Next</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>81</width>
               <height>23</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbtSend" >
             <property name="text" >
              <string>Send</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtReply" >
             <property name="text" >
              <string>Reply</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtForward" >
             <property name="text" >
              <string>Forward</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtChat" >
             <property name="text" >
              <string>Chat</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>wdtTabs</tabstop>
  <tabstop>lneSubject</tabstop>
  <tabstop>pbtNext</tabstop>
  <tabstop>pbtSend</tabstop>
  <tabstop>pbtReply</tabstop>
  <tabstop>pbtChat</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
